test/unit/math/mix/prob/multi_student_t_test.cpp

```diff
@@ -1,7 +1,62 @@
-#include <stan/math/mix.hpp>
-#include <gtest/gtest.h>
-#include <boost/random/mersenne_twister.hpp>
-#include <boost/math/distributions.hpp>
+#include <test/unit/math/test_ad.hpp>
+
+TEST(ProbDistributionsMultiStudentT, matvar) {
+  auto f
+      = [](const auto& y, const auto& nu, const auto& mu, const auto& sigma) {
+          auto sigma_sym = stan::math::multiply(0.5, sigma + sigma.transpose());
+          return stan::math::multi_student_t_lpdf(y, nu, mu, sigma_sym);
+        };
+
+  auto f_const_y = [](const auto& y) {
```

Comment:
What is the reason for these tests that make one argument constant?

Reply:
Four argument tests aren't available for `expect_ad`